setup.py

```diff
@@ -417,10 +418,11 @@ def get_extensions():
             "--ptxas-options=-O2",
             "--ptxas-options=-allow-expensive-optimizations=true",
         ]
-    elif torch.cuda.is_available() and torch.version.hip:
+    elif (torch.cuda.is_available() and torch.version.hip) or (
+        os.getenv("FORCE_ROCM", "0") == "1"
```

Comment:
Can we simply check if `HIP_ARCHITECTURES` is non-empty here? This would avoid an additional env variable
Also, does it make sense to build for ROCM if torch does not have hip?

Reply:
Well, we're trying to build on a CPU node, so I think `torch.cuda.is_available()` check fails. The wheels can hopefully used then on a machine with MI series GPUs. Good point about single env variable, we can do that